tests/subscriptions/test_data.py

```diff
@@ -91,29 +94,103 @@
     ),
 ]
 
+TESTS_OVER_SESSIONS = [
+    pytest.param(
+        DelegateSubscriptionData(
+            project_id=1,
+            query=(
+                "MATCH (sessions) "
+                "if(greater(sessions AS `_snuba_sessions`, 0), divide(sessions_crashed AS "
+                "`_snuba_sessions_crashed`, sessions AS `_snuba_sessions`), null AS `_snuba_null`) "
+                "AS `_snuba_crash_rate_alert_aggregate` "
+                "WHERE org_id = 1 AND project_id IN tuple(1) "
+                "LIMIT 1 "
+                "OFFSET 0 "
+                "GRANULARITY 3600"
+            ),
+            conditions=[],
+            aggregations=[
+                [
+                    "if(greater(sessions, 0), divide(sessions_crashed, sessions), null)",
+                    None,
+                    "crash_rate_alert_aggregate",
+                ]
+            ],
+            time_window=timedelta(minutes=120),
+            resolution=timedelta(minutes=1),
+            organization=1,
+        ),
+        None,
+        id="Delegate subscription",
+    ),
+    pytest.param(
+        LegacySubscriptionData(
+            project_id=1,
+            conditions=[],
+            aggregations=[
+                [
+                    "if(greater(sessions, 0), divide(sessions_crashed, sessions), null)",
+                    None,
+                    "crash_rate_alert_aggregate",
+                ]
+            ],
+            time_window=timedelta(minutes=120),
+            resolution=timedelta(minutes=1),
+        ),
+        JsonSchemaValidationException,
+        id="No organization provided for sessions dataset subscription",
+    ),
+]
+
 
-class TestBuildRequest(BaseSubscriptionTest):
+class TestBuildRequestBase:
```

Comment:
Why did you change the class inheritance here?

Reply:
Because `BaseSubscriptionTest` basically creates a bunch of dummy events in `setup_method` which I really don't need for the the crash rate alerts tests but I need functions like `compare_conditions`, and so I just subclass `BaseSubscriptionTest` when I am testing for events